UefiTestingPkg/Include/Protocol/MpManagement.h

```diff
@@ -0,0 +1,182 @@
+/** @file
+  When installed, the MP Management Protocol produces a collection of power
+  management service to power on/off the APs and suspend/resume all cores.
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#ifndef MP_MANAGEMENT_PROTOCOL_H_
+#define MP_MANAGEMENT_PROTOCOL_H_
+
+///
+/// Global ID for the MP_MANAGEMENT_PROTOCOL.
+///
+#define MP_MANAGEMENT_PROTOCOL_GUID \
+  { \
+    0x3fdda605, 0xa76e, 0x4f46, {0xad, 0x29, 0x12, 0xf4, 0x53, 0x1b, 0x3d, 0x08} \
+  }
+
+///
+/// Secretive combo to apply the operation to all APs.
+///
+#define OPERATION_FOR_ALL_APS  MAX_UINTN
+
+///
+/// Supported processor suspension states.
+///
+typedef enum {
+  AP_POWER_C1,
+  AP_POWER_C2,
+  AP_POWER_C3,
+  AP_POWER_NUM
+} AP_POWER_STATE;
+
+///
+/// Forward declaration for the MP_MANAGEMENT_PROTOCOL.
+///
+typedef struct _MP_MANAGEMENT_PROTOCOL MP_MANAGEMENT_PROTOCOL;
+
+typedef

```

Comment:
function header looks like its missing.

Reply:
This function should be removed. Good catch. Thanks.

Reply:
Removed in the latest update.